docs/redoc/master/openapi.json

```diff
@@ -621,11 +621,8 @@
                     "status": {
                       "type": "string",
                       "enum": [
-                        "ok"
+                        "created"
                       ]
-                    },
-                    "result": {
-                      "type": "boolean"
                     }
```

Comment:
We cannot change this for backwards compatibility reasons. We should keep `result = true`.

Reply:
thanks! wasn't sure about this as well